src/components/content/courses/CourseDetails.js

```
@@ -198,75 +202,188 @@ export default function CourseDetails({
+        <Divider className="my-4" />
+        
+        {/* Course details */}
+        <div className={`grid grid-cols-1 ${isPhone ? 'gap-4' : 'lg:grid-cols-3 gap-6'}`}>
+          {/* Left column: Description */}
+          <div className={`${isPhone ? '' : 'lg:col-span-2'}`}>
+            <h2 className="text-xl font-semibold mb-3 text-white">About This Course</h2>
+            <div className="text-gray-300 mb-4">
+              {processedEvent.description &&
+                processedEvent.description
+                  .split('\n')
+                  .map((line, index) => <p key={index} className={`${isPhone ? 'text-sm' : ''} mb-2`}>{line}</p>)}
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)